src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEFAULT_COLOR = "white";
+    private static final double DEFAULT_RADIUS = 10.0;
+    private static final int FIGURE_COUNT = 5;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public FigureSupplier(ColorSupplier colorSupplier) {
+        this.colorSupplier = colorSupplier;
+    }
+
```

Comment:
```suggestion
    private ColorSupplier colorSupplier = new ColorSupplier();
    private Random random = new Random();

```

You initialize at declaration, no need for the constructor, it shouldn't be different to Random